Extension/c_cpp_properties.schema.json

```diff
@@ -18,10 +18,7 @@
                     "compilerPath": {
                         "markdownDescription": "Full path of the compiler being used, e.g. `/usr/bin/gcc`, to enable more accurate IntelliSense.",
                         "descriptionHint": "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered.",
-                        "type": [
-                            "string",
-                            "null"
-                        ]
+                        "type": "string"
```

Comment:
I still see `null` as the default? Is that correct or not?

![image](https://github.com/user-attachments/assets/94914dd4-afe2-4fbe-bcc8-76da12b8e277)

Reply:
I think it is correct because we want to prevent `""` from being the default (and accidentally kept by users who expect the default values to be acceptable and not cause a major change in behavior).

Reply:
But as mentioned in the other message the c_cpp_properties.json version doesn't use the same defaults and it seems like it should?

Reply:
`null` exists for the C_Cpp.default.compilerPath setting as a way to "unset" a default value set at a higher scope. I did not remove that. However, `c_cpp_properties.json` does not have scopes so it does not need to support `null`.

Reply:
I just read your later comment, and will retract my statement above.